src/main/java/seedu/workbook/model/internship/Company.java

```diff
@@ -52,6 +53,21 @@ public boolean equals(Object other) {
                         && name.equals(((Company) other).name)); // state check
     }
 
+    public String toUpperCase(String companyName) {
```

Comment:
Could you add some javadocs to briefly explain this method and/or rename the method? Based on the method name I would think it changes the String to all uppercase, but I'm guessing there's something else being done if not you would have just used String.toUpperCase right

Reply:
I think i included the javadocs already. Any suggestions on what i can rename the method to?

Reply:
Maybe try `firstLetterOfEachWordToUpperCase()`